### PR TITLE
conversion to typescript

### LINKED ISSUE #17: Convert project to typescript

### PR DESCRIPTION
**Pull request by xsharonhe**
Converting everything (most things) to Typescript...

### CODE REVIEW

File: src/CodeEditor/CodeEditor.css [hunk 1]
```diff
@@ -1,14 +1,17 @@
 .ace_editor {
```
Comment (xsharonhe) [inline src/CodeEditor/CodeEditor.css:1]: the CSS for this module does not work? i couldnt figure out how to get it to work if anyone has inputs here

Comment (michelleshx) [reply src/CodeEditor/CodeEditor.css:1]: o i think it has to be renamed to be `CodeEditor.module.css` (and the import should be updated in the index.tsx file too) - lmk if that works!

Comment (michelleshx) [reply src/CodeEditor/CodeEditor.css:1]: got the css working if the import in index.tsx is changed from `import styles from "./CodeEditor.css";` to `import "./CodeEditor.css";`

File: setupProxy.ts
Comment (michelleshx) [inline setupProxy.ts:1]: not sure if this is a typescript issue but i couldnt get the proxy working for the 'ask george' when running locally?
image

**Review by michelleshx — CHANGES_REQUESTED** [review]
thank u for working on the conversion :pray: made a comment for the css fix for the code editor and the proxy file

File: src/CodeEditor/CodeEditor.css (hunk 1 above)
Comment (xsharonhe) [reply src/CodeEditor/CodeEditor.css:1]: this worked ty!

**Pull request merged**